lib/index.js

```diff
@@ -161,25 +161,6 @@ function PubSubClient(opts) {
 
 	this.timeout = opts.timeout || 10000;
 
-	let self = this;
-	if (opts.newrelic) {
-		let newrelic = opts.newrelic;
-		// on a retry, we are going to send a custom metric to newrelic if configured
-		this.on('retry', function (code, opts, retries, length) {
```

Comment:
@jareddlc what triggers this retry event that we can just toss this?

Reply:
a failed delivery, in my case it when i had an incorrect url, it would attempt to retry. arrow would merge newrelic config and the key property was undefined. I'm not sure if its a good idea for a public client try to connect to newrelic and send an error event.